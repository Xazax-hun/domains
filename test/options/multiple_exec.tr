// CMD: {args} {filename} --executions 3
init(50, 50, 0, 0);
translation(10, 0);
rotation(0, 0, 90);
translation(0, 10)
