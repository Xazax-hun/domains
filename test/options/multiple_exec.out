1. execution:
{ x: 50, y: 50 }
{ x: 60, y: 50 }
{ x: -50, y: 60 }
{ x: -50, y: 70 }
2. execution:
{ x: 50, y: 50 }
{ x: 60, y: 50 }
{ x: -50, y: 60 }
{ x: -50, y: 70 }
3. execution:
{ x: 50, y: 50 }
{ x: 60, y: 50 }
{ x: -50, y: 60 }
{ x: -50, y: 70 }
